packages/suite/src/views/wallet/settings/Container.ts

```diff
@@ -1,7 +1,7 @@
 import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
 import { injectIntl } from 'react-intl';
-
+import { goto } from '@suite-actions/routerActions';
```

Comment:
thought it is better to import is * as router actions :]

Reply:
good catch